plaso/parsers/plist_plugins/macuser.py

```diff
@@ -101,37 +97,26 @@ def GetEntries(self, parser_mediator, match=None, **unused_kwargs):
 
         shadow_hash_data = match.get('ShadowHashData', None)
         if date_time and isinstance(shadow_hash_data, (list, tuple)):
-          # Extract the hash password information.
-          # It is store in the attribute ShadowHasData which is
-          # a binary plist data; However biplist only extracts one
-          # level of binary plist, then it returns this information
-          # as a string.
-
-          # TODO: change this into a DataRange instead. For this we
-          # need the file offset and size of the ShadowHashData value data.
-          shadow_hash_data = shadow_hash_data[0]
-
-          resolver_context = context.Context()
-          fake_file = fake_file_io.FakeFile(
-              resolver_context, shadow_hash_data)
-          shadow_hash_data_path_spec = fake_path_spec.FakePathSpec(
-              location='ShadowHashData')
-          fake_file.open(path_spec=shadow_hash_data_path_spec)
-
+          # Extract the hash password information, which is stored in
+          # the attribute ShadowHasData which is a binary plist data.
```

Comment:
Typo here, ShadowHasData instead of ShadowHashData

Reply:
ack, will create a separate PR to fix